src/pat/inject/inject.js

```diff
@@ -305,6 +314,7 @@ define([
             if ((cfg.history === "record") && ("pushState" in history)) {
                 history.pushState({'url': cfg.url}, "", cfg.url);
             }
+            $target.trigger("pat-inject-content-loaded-final");
         },
```

Comment:
what happens with above event is also that it gets triggered for each image that gets loaded right?

could i use imagesLoaded jquery plugin when there are images to trigger only one ``pat-inject-content-loaded`` event? and in case there is no images we only trigger one ``pat-inject-content-loaded`` event as well.